src/config.rs

```diff
@@ -31,6 +32,14 @@ pub struct HmsConfig {
     pub high_priority: Option<bool>,
 }
 
+#[derive(Clone, Debug, Deserialize)]
+pub struct ThreemaGatewayConfig {
+    pub base_url: String,
+    pub identity: String,
+    pub secret: String,
+    pub private_key_file: String,
```

Comment:
There's a minor advantage in having a path to a file if the config leaks, obviously. Otherwise, I don't mind. If you don't have a strong opinion here, I'd like to leave it as is though because the SDKs also primarily use files and the same format.